index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body class="container">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
+      
+      <form id="survey-form">
+        <div class="input-container">
+          <div class="input-description">
+            <label id="name-label" for="name">Name</label>
+            <input required id="name" type="text" placeholder="Enter your name">
+          </div>
+          <div class="input-description">
+            <label id="email-label" for="email">Email</label>
+            <input required id="email" pattern=".+@globex\.com" type="email" placeholder="Enter your email">
+          </div>
```

Comment:
Sorry for this comment, something was bad here :smiling_face_with_tear:.
You've done well to include a new class for each element of the form, :dog:  but I would suggest using a section tag instead of a div tag because is more semantically correct. :smiley: